.github/workflows/wireless-performance-autotest.yml

```
@@ -70,6 +71,35 @@ jobs:
          shell: bash
          run: |
 
+           # extract site ID
+           SITE_ID=${{ matrix.json.site_id }}
+           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)